docs/.vitepress/config.mjs

```diff
@@ -486,8 +486,211 @@ export default defineConfig({
                             },
                         ]
                     },
-                ]
-            }
-        }
-    }
-})
+                ],
+            },
+        },
+        de: {
+            label: 'Deutsch',
+            lang: 'de-DE',
+            link: '/de',
+            themeConfig: {
+                editLink: {
+                    pattern:
+                        'https://github.com/hd-zero/hdzero-docs/edit/main/docs/:path',
+                },
+                nav: [{text: 'Dokumentation', link: '/'}],
+                sidebar: [
+                    {
+                        text: 'Home',
+                        items: [{text: 'Was ist HDZero?', link: '/'}],
+                    },
+                    {
+                        text: 'VTX',
+                        items: [
+                            {
+                                text: 'HDZero VTX Produkte',
+                                link: '/vtx-summary',
+                                collapsed: true,
+                                items: [
+                                    {text: 'Whoop V2 VTX', link: '/whoop-v2'},
```

Comment:
You can refer to the Russian and Chinese configurations; a prefix should be added before the path.